cmd/kafkarator/main.go

```diff
@@ -215,6 +216,12 @@ func primary(quit QuitChannel, logger *log.Logger, mgr manager.Manager, cryptMan
 		return
 	}
 
+	generator, err := certificate.NewExecGenerator()
```

Comment:
It will be a problem that the store password changes between Kafkarator restarts. When credentials are rotated, applications can rehash files from disk to get the new credentials, but environment variables are not updated until the application is restarted.

Maybe just use `changeme` instead?

Reply:
> Maybe just use `changeme` instead?

Fine, I'll relent 😄